reana_client/utils.py

```diff
@@ -173,6 +180,49 @@ def _prepare_kwargs(reana_yaml):
         raise e
 
 
+def _validate_environment_tags(reana_yaml):
+    """Validate environments in REANA specification file according to workflow type.
+
+    :param reana_yaml: Dictionary which represents REANA specifications file.
+    :return: True if environment tags are valid.
```

Comment:
```suggestion
    :return: True if environment is valid.
```

Reply:
I removed the `return`, we don't need this now. Removed it.